core/src/parser.rs

```diff
@@ -593,11 +590,18 @@ fn get_field_decorators(attrs: &[syn::Attribute]) -> HashMap<SupportedLanguage,
                 language,
                 list.iter()
                     .flat_map(|nested| match nested {
-                        NestedMeta::Meta(Meta::Path(path)) => path.get_ident(),
+                        NestedMeta::Meta(Meta::Path(path)) => {
```

Comment:
Recommend being more strict with the matching / acceptance here. Currently, this will accept things like `typeshare(a::b::c)` and return them as `FieldDecorator::Word("c")`.